gpbft/gpbft.go

```diff
@@ -1281,16 +1308,34 @@ func (c *convergeState) FindMaxTicketProposal(table PowerTable) ConvergeValue {
 			}
 		}
 	}
+
+	// Check if self participant has entered CONVERGE phase.
```

Comment:
I think we can focus the documentation here a little, but the code looks good.

I would say that if the node has received any converged messages, it finds the best ticket from those. That will include the node's own messages, once they are delivered asynchronously. Fall back to the self-value if nothing has been received yet. Note that this introduces a brief possibility of a node ignoring its own value if it has not received the ticket for it yet.

I don't think we need to talk about checks on whether the phase has started (goes with renaming the method) or make statements about what is assumed to have been sent down at this level. This is a change in focus of the comments from what other code is assumed to have done to direct statements about what this code does.